HACKING.md

```diff
@@ -102,9 +102,19 @@ Install playwright and its browsers
 
     npx playwright install
 
-The tests expect the environment on localhost to be accessible. Execute `dotrun` and start tests with
+The e2e tests can be run against the LTS version (v5.0) or the latest release version for LXD. If you want to run the tests against the LTS version, first downgrade your lxd with
 
-    npx playwright test
+    snap refresh lxd --channel 5.0/stable
```

Comment:
Maybe suggest to use `latest/edge` here, so the next command below `test-e2e-edge` is connected?